deploy/handler/operator.yaml

```diff
@@ -1,37 +1,35 @@
 {{define "handlerPrefix"}}{{with $prefix := .HandlerPrefix}}{{$prefix | printf "%s-"}}{{end -}}{{end}}
 ---
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
```

Comment:
Q: We want the webhook pods to run on Masters only because of performance (k8s-api run on masters), right?

Reply:
Is more about stability than performance, users will configure networking usually at workers so workers are going to be more instable alas breaking the webhook if we run it there also if you think about it the webhook is kind of knmstate control-plane so it belongs to masters.